lazygit crashes on launch

**Describe the bug**
lazygit just crashes when it's launched in one of my repos. Other repos seem to work.

It seems to be an array index bug, maybe due to files with umlauts in their filename?
It worked on previous versions, I think up until 0.14.x or so.

Here is the message:
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/jesseduffield/lazygit/pkg/commands.(*BranchListBuilder).obtainBranches(0xc000393998, 0x15, 0xc000393860, 0x55b0150ccb7e)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/commands/branch_list_builder.go:54 +0x60c
github.com/jesseduffield/lazygit/pkg/commands.(*BranchListBuilder).Build(0xc000393998, 0xc000393aa0, 0xc0000c94a0, 0xc0003dff00)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/commands/branch_list_builder.go:95 +0x45
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).refreshBranches.func1(0xc00013e000, 0xc0003939e4, 0x3)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/gui/branches_panel.go:70 +0x54
github.com/jesseduffield/gocui.(*Gui).consumeevents(0xc00013e000, 0xc000393b28, 0x2)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:483 +0x1ea
github.com/jesseduffield/gocui.(*Gui).MainLoop(0xc00013e000, 0xc000000003, 0xc000393ca0)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:465 +0x217
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).Run(0xc000138000, 0x0, 0x0)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:1033 +0x608
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).RunWithSubprocesses(0xc000138000, 0x0, 0x0)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:1042 +0x42
github.com/jesseduffield/lazygit/pkg/app.(*App).Run(0xc00025ad90, 0xc00018ea00, 0xc00025ad90)
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/pkg/app/app.go:158 +0x6d
main.main()
        /home/sed/.cache/yay/lazygit-git/src/src/github.com/jesseduffield/lazygit/main.go:67 +0x75c

**Expected behavior**
lazygit doesn't crash ;-)

**Desktop (please complete the following information):**
 commit=2d8ed5e2, build date=20200327.081646, build source=unknown, version=v0.18.2d8ed5e2, os=linux, arch=amd64

**Additional context**
It worked on previous versions, I think up until 0.14.x or so.

--- a/pkg/commands/commit_list_builder.go
+++ b/pkg/commands/commit_list_builder.go
@@ -109,11 +109,12 @@ func (c *CommitListBuilder) GetCommits(options GetCommitsOptions) ([]*Commit, er
 	cmd := c.getLogCmd(options)
 
 	err = RunLineOutputCmd(cmd, func(line string) (bool, error) {
-		commit := c.extractCommitFromLine(line)
-		_, unpushed := unpushedCommits[commit.ShortSha()]
-		commit.Status = map[bool]string{true: "unpushed", false: "pushed"}[unpushed]
-		commits = append(commits, commit)
-
+		if strings.Split(line, " ")[0] != "gpg:" {
+			commit := c.extractCommitFromLine(line)
+			_, unpushed := unpushedCommits[commit.ShortSha()]
+			commit.Status = map[bool]string{true: "unpushed", false: "pushed"}[unpushed]
+			commits = append(commits, commit)
+		}
 		return false, nil
 	})
 	if err != nil {

